ssl/statem/extensions_srvr.c

```diff
@@ -780,7 +780,7 @@ int tls_parse_ctos_psk(SSL *s, PACKET *pkt, unsigned int context, X509 *x,
      * client's due to the network latency). Therefore we add 1000ms to our age
      * calculation to adjust for rounding errors.
      */
-    if (sess->timeout >= agesec
+    if ((uint32_t)sess->timeout >= agesec
```

Comment:
While it doesn't really matter, *formally* it's probably more appropriate to cast agesec. Because sess->timeout is more likely to be wider than agesec. In other words widening over truncating. See even #3105...